enterprise/app/jobs/account_sla_job.rb

```diff
@@ -0,0 +1,9 @@
+class AccountSlaJob < ApplicationJob
+  queue_as :medium
+
+  def perform(account)
+    AppliedSla.includes(:conversation, :sla_policy).where(account_id: account.id, sla_status: 'active').each do |applied_sla|
```

Comment:
@sojan-official: I used include here to do rails eager loading to avoid N+1 queries. Otherwise, for every `applied_sla`, we would need to fetch conversation and sla_policy separately in `evaluate_sla_service.rb` with two separate queries.